config/initializers/devise.rb

```diff
@@ -21,7 +21,8 @@
   # available as additional gems.
   require 'devise/orm/active_record'
   require 'omniauth-linkedin'
-  config.omniauth :linkedin, ENV["APP_ID"], ENV["APP_SECRET"]
+  config.omniauth :linkedin, ENV["APP_ID"], ENV["APP_SECRET"],
+  :client_options => {:ssl => {:ca_path => '/usr/lib/ssl/certs/ca-certificates.crt'}}
```

Comment:
Line is too long. [85/80]<br>Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.